### PR TITLE
Upgrade the SMS client with Github Actions

### PR DESCRIPTION
**Pull request by avinashbr123**
Removed appveyor.yml and build.ps1
Added Github-cicd.yml and updated SDK version

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (smahapatra86) [inline .gitattributes:1]: What are the changes in this file? Is it related to Github actions?

File: .github/workflows/github-cicd.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+name: Sms Client NuGet CI
+
+env:
+  PROJPATH: './src/Kmd.Logic.Sms.Client'
+  DOTNET_VERSION: '5.0.x'
```
Comment (smahapatra86) [inline .github/workflows/github-cicd.yml:5]: is the client project `.net core 3.1` or `5.0`?

File: .github/workflows/github-cicd.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - name: Pack - PreRelease
+      run: dotnet pack '${{ env.PROJPATH }}/Kmd.Logic.Sms.Client.csproj' --output ${{ env.PROJPATH }}/nupkgs --no-restore -p:VersionSuffix=dev-${{ steps.current-time.outputs.formattedTime }}
+      if: GITHUB.REF != 'refs/heads/master'
+
+    - name: Pack - Release
+      run: dotnet pack '${{ env.PROJPATH }}/Kmd.Logic.Sms.Client.csproj' --output ${{ env.PROJPATH }}/nupkgs --no-restore
+      if: GITHUB.REF == 'refs/heads/master'
+
+    - name: Publish NuGet
+      run: dotnet nuget push ${{ env.PROJPATH }}/nupkgs/Kmd.Logic.Sms.Client*.nupkg --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
```
Comment (smahapatra86) [inline .github/workflows/github-cicd.yml:53]: May need to check if will work or not.  There is no secret added at sms repository level. But I believe this secret will be taken from root level. Let's see.

File: .gitattributes (hunk 1 above)
Comment (avinashbr123) [reply .gitattributes]: Yes removed this file is not part of Github actions

**Review by smahapatra86 — APPROVED** [review]

**Review by avin101 — COMMENTED** [review]
Please validate if the sample works fine with the changes done to the client.

**Comment by avinashbr123** [comment]
> Please validate if the sample works fine with the changes done to the client.



> Please validate if the sample works fine with the changes done to the client.

Yes, tested working fine

**Review by avin101 — APPROVED** [review]

**Pull request merged**